docs/setup/betamocks.md

```diff
@@ -26,8 +26,9 @@ without being connect to the VA VPN. By default all users have been mocked for M
 only M. Webb (vets.gov.user+228@gmail.com) will work for the other services unless their mock data has been added.
 
 ## Mocking a Service
-1. Add the Betamocks middleware to the service to be mocked. It should
-be the first response middleware listed in the connection block.
+1. Add the Betamocks middleware to the service to be mocked. Previously, it was recommended that it should be
```

Comment:
I'd opt to leave out the historical approach in a setup-type document, IMO it leads to more confusion.

```suggestion
If a service class implements response middleware, it is important to consider the order in which the middleware is stacked. For further details, refer to the [Faraday API documentation](https://www.rubydoc.info/gems/faraday#Advanced_middleware_usage). 

In the following example, Betamocks will only record the raw response from the backing service, and will not record any transformations applied by the `::FacilityParser` or `::FacilityValidator` middlewares.  

```ruby
def connection
  Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
    conn.use :breakers
    conn.request :json
    
    conn.response :raise_error, error_prefix: service_name
    conn.response :facility_parser
    conn.response :facility_validator
    conn.response :betamocks if Settings.locators.mock_gis
    
    conn.adapter Faraday.default_adapter
  end
end
```
```